src/metabase/sync/task/sync_databases.clj

```diff
@@ -292,7 +292,8 @@
                      (:name task-info)
                      (:name database)
                      (cron-schedule database task-info)))
-        (task/add-trigger! new-trigger))
+        ;; ffffffffuuuuuu
+        #_(task/add-trigger! new-trigger))
```

Comment:
I don't know if its a code or database thing, but I'm getting tons of errors in my REPL from this, ever since I got back from leave. I'll back this stuff out before merging, just gave up on debugging it for now.

Reply:
what error do you get?

Reply:
I get NPEs triggered by toucan hooks. Will create a new database and see if that fixes things.